src/main/java/com/softserve/academy/spaced/repetition/logger/logs/log.txt

```diff
@@ -1 +1,59 @@
 
```

Comment:
Please remove log file from repository
https://stackoverflow.com/questions/7927230/remove-directory-from-remote-repository-after-adding-them-to-gitignore